css-four-column-layout/styles.css

```diff
@@ -0,0 +1,44 @@
+.container {
+  width: 1400px;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: "Barlow Condensed", sans-serif;
+  font-family: "Changa One", cursive;
+  font-family: "Delicious Handrawn", cursive;
+  font-family: Roboto, sans-serif;
+}
+
+img {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.column-fourth-two > img {
+  width: 20%;
+  padding-left: 130px;
+}
+
+.column-4 {
+  width: 100%;
+  justify-content: center;
+  padding-left: 110px;
+}
```

Comment:
If these are supposed to be columns of 4, why are you setting the `width` to 100%? Shouldn't this be `flex-basis: 25%`?